src/sparseml/onnx/base.py

```diff
@@ -31,20 +31,24 @@
     import onnxruntime
 
     onnxruntime_err = None
+    onnxruntime_gpu_installed = onnxruntime.get_device() == "GPU"
 except Exception as err:
     onnxruntime = object()  # TODO: populate with fake object for necessary imports
     onnxruntime_err = err
-
+    onnxruntime_gpu_installed = False
 
 __all__ = [
     "onnx",
     "onnx_err",
     "onnxruntime",
     "onnxruntime_err",
+    "onnxruntime_gpu_installed",
```

Comment:
I'm not sold on the additional checks for a specific provider type. ORT has a lot of providers that we may want to support in the future, can we make something more generic here for checking for a given provider or list of providers?